angular.json

```diff
@@ -32,12 +32,12 @@
             "aot": true,
             "assets": ["src/assets"],
             "scripts": [],
-            "styles": ["src/styles/themes/default/style.scss"],
+            "styles": ["src/styles/themes/b2b/style.scss"],
             "stylePreprocessorOptions": {
-              "includePaths": ["src/styles/themes/default"]
+              "includePaths": ["src/styles/themes/b2b"]
```

Comment:
I would honestly leave this at "default" or rename it "placeholder" to showcase that is will be replaced by the build system. 🤔

Same for the other default paths. Changing the`defaultConfiguration`s is instead mandatory.

Reply:
Changed the `stylePreprocessorOptions` to use `placeholder` instead of `b2b`.

Reply:
@dhhyi did I find the right places?